src/components/language-selector/language-selector.css

```diff
@@ -18,6 +18,6 @@
     user-select: none;
     outline: none;
     background: rgba(255, 255, 255, 0.2);
-    color: white;
+    color: black;
```

Comment:
Black doesn't look great when everything else on the menu bar is white. 

Could you switch the color to #3373cc (our dark blue), increase the contrast on the background (try .5 instead of .2)

The grey icon probably won't look good next to the blue, and we really don't need it right now, so let's just remove it. 

Eventually we'll be doing a real design pass for the language selection, and this will be completely different.